### ISSUE TITLE
Error in component.d.ts after Importing the ngx-skeleton-loader to app.module.ts in Angular 14.2.7

### ISSUE THREAD

**Issue by reneskarabot** [willmendesneto/ngx-skeleton-loader#140]
**I'm submitting a ...** (check one with "x")

- [x] bug report => search github for a similar issue or PR before submitting
- [ ] feature request

**Current behavior**

After install the ngx-skeleton-loader using the: 
and add it to app.module.ts file in my Angular 14.2.7 project: import { NgxSkeletonLoaderModule } from 'ngx-skeleton-loader';

I got this error:

Error: node_modules/ngx-skeleton-loader/lib/ngx-skeleton-loader.component.d.ts:22:18 - error TS2707: Generic type 'ɵɵComponentDeclaration' requires between 7 and
8 type arguments.

22     static ɵcmp: i0.ɵɵComponentDeclaration<NgxSkeletonLoaderComponent, "ngx-skeleton-loader", never, { "count": "count"; "loadingText": "loadingText"; "appearance": "appearance"; "animation": "animation"; "ariaLabel": "ariaLabel"; "theme": "theme"; }, {}, never, ["*"], false, never>;

**Expected behavior**

Application compiled without errors.

**Please tell us about your environment:**

    "@angular/animations": "^14.2.0",
    "@angular/cdk": "^14.2.7",
    "@angular/common": "^14.2.0",
    "@angular/compiler": "^14.2.0",
    "@angular/core": "^14.2.0",
    "@angular/forms": "^14.2.0",
    "@angular/material": "^14.2.7",

    "ngx-skeleton-loader": "^6.0.0"   --- Same issue with v: 7.0.0 and 5.0.0

**Comment by willmendesneto** [comment]
Hi @reneskarabot , how are you?

Could you please try to simulate your issue an issue and simulate it in a Stackblitz Demo and share the link in here, please? I tried to simulate it, but I wasn't able to.

https://stackblitz.com/edit/ngx-skeleton-loader-sample?file=src%2Fapp%2Fapp.component.ts

**Comment by reneskarabot** [comment]
Hi @willmendesneto,

Thank you for getting back to me so quickly.

I am not quite familiar with Stackblitz – how to select the specific angular version etc… So I did the following:
1.	**Prepare a new development environment from scratch**
2.	**Install Node.js** (version 18.14.0)
3.	**NPM** (version 9.4.2)
4.	**Angular CLI** (version 14.2.4) – the same version I am using in my bigger project
5.	**Create a new angular app**
6.	**Install the ngx-skeleton-loader using the Visual Studio Code terminal**: npm install ngx-skeleton-loader –save
7.	**Add the following import to app.module.ts**: import { NgxSkeletonLoaderModule } from 'ngx-skeleton-loader';
8.	**Save the project (ng serve compile it automatically)** and I got the same error as in my bigger project: 

_Build at: 2023-02-08T10:53:25.552Z - Hash: 94123b1be2c06f66 - Time: 3237ms

Error: node_modules/ngx-skeleton-loader/lib/ngx-skeleton-loader.component.d.ts:22:18 - error TS2707: Generic type 'ɵɵComponentDeclaration' requires between 7 and 8 type arguments.

22     static ɵcmp: i0.ɵɵComponentDeclaration<NgxSkeletonLoaderComponent, "ngx-skeleton-loader", never, { "count": "count"; "loadingText": "loadingText"; "appearance": "appearance"; "animation": "animation"; "ariaLabel": "ariaLabel"; "theme": "theme"; }, {}, never, ["*"], false, never>;_

Image of error: https://drive.google.com/file/d/1-tWAiTLaVHrlcbZYpO8A9UX61eU4b9wB/view?usp=share_link

I uploaded the code of the clean test project where the error occurred to my Github repo, available here: https://github.com/reneskarabot/skeleton-loading-test.git

If you can try it on your side with the same setup and see if you get the same error.

I don't know if there is something wrong with npm-skeleton-loader library for my current setup, or if I am doing something wrong, but I don't see it.

All your efforts are highly appreciated.

Thank you very much,
Rene

**Comment by reneskarabot** [comment]
Hi @willmendesneto ,

I tried to update the angular to the latest version:

@angular-devkit/core            15.1.5
@angular-devkit/schematics      15.1.5
@angular/cli                    15.1.5


And ngx-skeleton-loader is working without problem.

So it looks like there is something wrong with ngx-skeleton-loader when the following version is used:
@angular-devkit/core         14.2.4 (cli-only)
@angular-devkit/schematics   14.2.4 (cli-only)
@angular/cli           14.2.4 

Is it possible to somehow fix the ngx-skeleton-loader to work with 14.2.4 version? This is the version I am using in my bigger project and at the moment I cannot update the angular to the latest version in this project where I want to user ngx-skeleton-loader.

I uploaded the code of the clean test project created with 14.2.4 version, where the error occurred to my Github repo, available here: https://github.com/reneskarabot/skeleton-loading-test.git

Best regards,
Rene

**Comment by willmendesneto** [comment]
@reneskarabot Could you give me access for the repo https://github.com/reneskarabot/skeleton-loading-test.git, please? I'm trying to access the link, but I'm going to Github's 404 page

**Comment by reneskarabot** [comment]
Hi @willmendesneto,

Sorry for that I thought that I marked the repo as public, but sure I did not.

Try now, it should work I change it to public.

https://github.com/reneskarabot/skeleton-loading-test.git

Best regards,
Rene

**Comment by willmendesneto** [comment]
@reneskarabot Looks like there's some misconfiguration in your project:

- The issue is solved by downgrading the library to `ngx-skeleton-loader@6.0.0` and using Node v14.20.0 locally. No issues so far.
- Your package-lock.json is malformed, solved by running `rm -rf node_modules && npm install`
- I added the module and a skeleton. Working as expected


Saying that, the issue is related to some local misconfiguration. Closing this issue now. thanks @reneskarabot !
